### DISCUSSION TITLE
Looking to only select one row (disable multi-row-select)

### DISCUSSION THREAD

**Discussion by chrisorr0264** [Akascape/CTkTable#78] [Q&A]
Hi there.

Thanks for the app...it is very useful.

I am needing help in having the ability for a row on the table to be selected (clicking any column on the row), that would "get" the data from the row.  (I have this working).

But I want to allow the users to select a different row that would: 1. unhighlight the first row and 2. return the data for the second row.  Right now it seems like selecting multiple rows creates a list of the values of each row.

Is there a way to disable the selection of multiple rows?

Thanks,

Chris

**Comment by 3JloDey** [comment]
```
def select_row(cell: dict) -> None:
    global selected_row
    current_row = cell["row"]

    if len(selected_row) == 0:
        table.select_row(current_row)
        selected_row.append(current_row)

    elif current_row == selected_row[-1]:
        table.deselect_row(selected_row.pop())

    else:
        table.deselect_row(selected_row.pop())
        table.select_row(current_row)
        selected_row.append(current_row)
```
